python_modules/libraries/dagster-shared/setup.py

```diff
@@ -39,6 +39,7 @@ def get_version() -> str:
         "packaging>=20.9",
         "pydantic>=2,<3.0.0",
         "typing_extensions>=4.10.0,<5",
+        "tomlkit",
```

Comment:
this results in a new dep in dagster core right? is this particularly heavy / did dagster core have this before?

Reply:
It's been a dep for a few weeks already, it just wasn't encoded in setup.py